Casks/r/restapia.rb

```diff
@@ -0,0 +1,26 @@
+cask "restapia" do
+  arch arm: "arm64"
+
+  version "0.8.12"
+  sha256 "d2996269fc44aa86d26d79fcdceaf6c29a095b2370aee04acf48dd1b03946eb0"
+
+  url "https://github.com/RestApia/Releases.Osx.Stable/releases/download/v#{version}-osx-stable/RestApia-osx-stable-Setup.pkg",
+      verified: "github.com/RestApia/Releases.Osx.Stable/releases/download/"
+  name "RestApia"
+  desc "HTTP API client offering flexible, code-based request definition"
```

Comment:
```suggestion
  desc "HTTP API client"
```

Per our [desc stanza guidelines](https://docs.brew.sh/Cask-Cookbook#stanza-desc), we trim the marketing fluff out of descriptions in `homebrew-cask`.